CodeEditModules/Modules/Keybindings/src/KeybindingManager.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  KeybindingManager.swift
+//  
+//
+//  Created by Alex on 09.05.2022.
+//
+
+import Foundation
+import SwiftUI
+
+// swiftlint:disable missing_docs
```

Comment:
I see there's documentation for `KeybindingManager` in the DocC catalog. Please add documentation for public vars, lets, as well as methods none-the-less. Also when disabling the swiftlint warning, use `swiftlint:disable:next missing_docs` to only apply this to the next line of code instead of the entire file.